KoAR.Core/ItemBuffMemory.cs

```diff
@@ -30,10 +30,10 @@ private static class Offsets
                 
         public const int MaxItemBuffs = 7;
 
-        internal ItemBuffMemory(int coreOffset, int coreLength)
+        internal ItemBuffMemory(GameSave gameSave, int coreOffset, int coreLength)
```

Comment:
can we just pass the byte array to these ctors they don't really care about the game save at all.